index.html

```diff
@@ -0,0 +1,91 @@
+        <p id='subm'><input type="submit" id='submit' name="submit" value="GET STARTED"></p>
+    </form>
+    <div id='details'>
+        <label for="prop1" class="proplab">Premium Materials</label>
+        <p id='prop1'>
+            Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your
+            purchase.</p>
+        <label for="prop2" class="proplab">Fast Shipping</label>
+        <p id='prop2'>
+            We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns
+            if you are not satisfied.</p>
+        <label for="prop3" class="proplab">Quality Assurance</label>
+        <p id='prop3'>
+            For every purchase you make, we will ensure there are no damages or faults and we will check and test the
+            pitch of your instrument.</p>
+    </div>
+    <div id='boxs'>
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    <div class='mytable'>
+        <table id='pricing' class='table'>
+            <tr>
+                <th>TENOR TROMBONE</th>
+            </tr>
+            <tr>
+                <td id='det'>
+                    <h1>600$</h2> Lorem ipsum.
+                        Lorem ipsum.
+                        Lorem ipsum dolor.
+                        Lorem ipsum.<br><br><br><input type="submit" id='select' name="submit" value="SELECT">
+                </td>
+            </tr>
+        </table>
+        
+        <table id='pricing2' class='table'>
+            <tr>
+                <th>BASS TROMBONE</th>
+            </tr>
+            <tr>
+                <td id='det'>
+                    <h1>900$</h2> Lorem ipsum.
+                        Lorem ipsum.
+                        Lorem ipsum dolor.
+                        Lorem ipsum.<br><br><br><input type="submit" id='select' name="submit" value="SELECT">
+                </td>
+            </tr>
+        </table>
+        <table id='pricing3' class='table'>
+            <tr>
+                <th>VALVE TROMBONE</th>
+            </tr>
+            <tr>
+                <td id='det'>
+                    <h1>1200$</h2> Lorem ipsum.
+                        Lorem ipsum.
+                        Lorem ipsum dolor.
+                        Lorem ipsum.<br><br><br><input type="submit" id='select' name="submit" value="SELECT">
+                </td>
+            </tr>
+        </table>
+        </div>
+    </div>
+    <footer id="footer">
+        <a href="#" class='foot-link'>Privacy</a>
+        <a href='#' class='foot-link'>Terms</a>
+        <a href="#" class='foot-link'>Contact</a>
+        <p id='copyright'>Copyright 2016, Original Trombones</p>
+    </footer>
+</body>
+</html>
+<My First Project>
```

Comment:
- [ ] [OPTIONAL] This is an invalid HTML tag. It would be better if it's removed to make your code neater and so it won't cause any problem in your project.